pontoon/contributors/static/js/settings.js

```diff
@@ -1,4 +1,118 @@
 $(function () {
+  $('.field .input')
+    .unbind('keydown.pontoon')
+    .bind('keydown.pontoon', function (e) {
+      if (e.which === 13) {
+        $(this).trigger('blur');
+        return false;
+      }
+    });
+
+  $('.field .input').on('blur', function (e) {
+    e.preventDefault();
+
+    const self = $(this);
+    const value = self.val().trim();
+    const attribute = self.data('attribute');
+    const originalValue = self.data('original-value') || '';
```

Comment:
What's the reason for using `self.data` rather than `this.dataset`? Values in the latter are automatically reflected in the element attributes, and so wouldn't need the duplicate setters that you have below.

In general, where the functionality overlaps, we should prefer DOM over jQuery.